analyzer/src/main/kotlin/managers/SpdxDocumentFile.kt

```diff
@@ -113,7 +113,16 @@ private fun SpdxPackage.getRemoteArtifact(): RemoteArtifact? =
     when {
         SpdxConstants.isNotPresent(downloadLocation) -> null
         SPDX_VCS_PREFIXES.any { (prefix, _) -> downloadLocation.startsWith(prefix) } -> null
-        else -> RemoteArtifact(downloadLocation, Hash.NONE)
+        else -> {
+            if (downloadLocation.endsWith(".git")) {
+                log.warn {
+                    "The download location $downloadLocation of SPDX package '$spdxId' looks like a Git repository " +
+                            "URL but it lacks the 'git+' prefix and thus will be treated as an artifact URL."
+                }
+            }
+
+            RemoteArtifact(downloadLocation, Hash.NONE)
+        }
```

Comment:
It would require a bigger refactoring, but I think it would be very useful to also create an `OrtIssue` for this, because it will make the problem more visible for users.

Reply:
I absolutely agree. However, the bigger factoring was the reason why I didn't do any better than the current proposal. In the end, it would be nice if an "issues registry" would be available "everywhere" via DI or something, so `OrtIssue`s could be created anytime from anywhere...

Reply:
Indeed, but that would be an even bigger refactoring.
I found one commit message nit: Typo in "locations".